contracts/SpokePool.sol

```diff
@@ -1271,10 +1274,27 @@ abstract contract SpokePool is
             relayerRefundLeaf.leafId,
             relayerRefundLeaf.l2TokenAddress,
             relayerRefundLeaf.refundAddresses,
+            deferredRefunds,
             msg.sender
         );
     }
 
+    /**
+     * @notice Enables a relayer to claim outstanding repayments. Should virtually never be used, unless for some reason
+     * relayer repayment transfer fails for reasons such as token transfer reverts due to blacklisting. In this case,
+     * the relayer can still call this method and claim the tokens to a new address.
+     * @param l2TokenAddress Address of the L2 token to claim refunds for.
+     * @param refundAddress Address to send the refund to.
+     */
+    function claimRelayerRefund(address l2TokenAddress, address refundAddress) public {
+        uint256 liability = relayerRefund[l2TokenAddress][msg.sender];
+        require(liability > 0, "No liability to claim");
```

Comment:
nit: rest of the contract uses if/revert pattern instead of reverts. Also in other places we use custom errors

Reply:
we use `require` syntax in many other files but you're right that in this file we don't. I've updated it accordingly.